service/notifications/notifications.go

```diff
@@ -2,23 +2,24 @@ package notifications
 
 import (
 	"github.com/weaveworks/flux/history"
+	"github.com/weaveworks/flux/service"
 	"github.com/weaveworks/flux/service/instance"
 )
 
 var DefaultNotifyEvents = []string{"release", "autorelease"}
 
-func Event(cfg instance.Config, e history.Event) error {
+func Event(cfg instance.Config, e history.Event, instanceID service.InstanceID) error {
```

Comment:
I think it would be a small improvement here to put this `InstanceID` inside the `instance.Config`.
(but see also later comments about including it in HookURL)